### ISSUE TITLE
Reapplying a key fails when key was manually removed

### ISSUE THREAD

**Issue by rbozan** [pauldn-wttj/terraform-provider-meilisearch#102]
Hi, thanks for the package.

When you create a key in Meilisearch using the provider, afterwards manually remove the keys (not through Terraform) and try to reapply Terraform, you would get the following errors:

```
╷
│ Error: Error Reading Meilisearch Key
│ 
│   with meilisearch_key.admin,
│   on meilisearch.tf line 5, in resource "meilisearch_key" "admin":
│    5: resource "meilisearch_key" "admin" {
│ 
│ Could not read Meilisearch key ID *hidden**: unaccepted status code found: 404 expected: [200], MeilisearchApiError Message: API key
│ `*hidden**` not found., Code: api_key_not_found, Type: invalid_request, Link: https://docs.meilisearch.com/errors#api_key_not_found (path "GET
│ /keys/*hidden*" with method "GetKey")
╵
╷
│ Error: Error Reading Meilisearch Key
│ 
│   with meilisearch_key.search,
│   on meilisearch.tf line 13, in resource "meilisearch_key" "search":
│   13: resource "meilisearch_key" "search" {
│ 
│ Could not read Meilisearch key ID *hidden*: unaccepted status code found: 404 expected: [200], MeilisearchApiError Message: API key
│ `*hidden*` not found., Code: api_key_not_found, Type: invalid_request, Link: https://docs.meilisearch.com/errors#api_key_not_found (path "GET
│ /keys/*hidden*" with method "GetKey")
╵
```

I would expect the creation to occur again instead of needing to manually remove the state using `terraform state rm`.

**Comment by pauldn-wttj** [comment]
Hello @rbozan :wave:

Thanks for the issue and the process to reproduce it, I'll look into that when I get the chance in the next days! :slightly_smiling_face:

**Comment by pauldn-wttj** [comment]
I merged a fix (#104) to ensure recreation of the key if it was manually removed. It was released under [`v0.0.3`](https://github.com/pauldn-wttj/terraform-provider-meilisearch/releases/tag/v0.0.3).

Thanks for the feedback, I'm closing this issue, feel free to re-open it if you still encounter the issue. :slightly_smiling_face:
